### DISCUSSION TITLE
V3 and user's tenant

### DISCUSSION THREAD

**Discussion by FSElias** [filamentphp/filament#7668] [Help] [answered]
Hi, please what to do with this error?
The model [App\Models\User] does not have a relationship named [team]. You can change the relationship being used by passing it to the [ownershipRelationship] argument of the [tenant()] method in configuration. You can change the relationship being used per-resource by setting it as the [$tenantOwnershipRelationshipName] static property on the [App\Filament\Admin\Resources\UserResource] resource class.

**Comment by danharrin** [comment]
I'm sorry but please read it. It literally tells you what to do.

**Reply by FSElias** [reply]
@danharrin 
Sorry for not expressing myself correctly, I have a ManyToMany relationship between User and Team models, so what to put in "team" relationship of the currently logged-in user?

Tried to return return Filament::getTenant()
but got this error:
Filament\Resources\Resource::getTenantOwnershipRelationship(): Return value must be of type Illuminate\Database\Eloquent\Relations\Relation, App\Models\Team returned

**Comment by FSElias** [comment, marked as answer]
Added this function to the User model and worked, does this approach have a downside?

```php
public function team()
{
    return $this->teams()->where('team_id', Filament::getTenant()->id);
}

```

**Reply by boralp** [reply]
Also adding a return type might be better

```
use Illuminate\Database\Eloquent\Relations\BelongsToMany;

public function team(): BelongsToMany {}
```
